### ISSUE TITLE
Dropdown arrow/caret not rendering properly in IE9

### ISSUE THREAD

**Issue by trebiano** [twbs/bootstrap#16203]
I've noticed that the little arrow of dropdown menus in Bootstrap's navbar doesn't render properly in InternetExplorer 9 (Windows Vista)...

Have a look to the screenshot below:
![sp5lqe](https://cloud.githubusercontent.com/assets/419884/7038882/0065e538-dd6d-11e4-8734-034fb3570a7f.jpg)

from the official Bootstrap's example page:
~~http://getbootstrap.com/examples/theme/~~
http://getbootstrap.com/examples/sticky-footer-navbar/

Anyone experimenting the same weird thing ? Any way to adjust it ?

Thanks

**Comment by zacechola** [comment]
Potentially related to #16172.

**Comment by cvrebert** [comment]
I am unable to reproduce this on Sauce Labs, Win 7, IE 9:
![0002screenshot](https://cloud.githubusercontent.com/assets/419884/7038954/470801d2-dd6e-11e4-8f5a-b4b7a03d16e3.png)

**Comment by trebiano** [comment]
Thanks for your replies... don't know exactly what has changed in the meantime, but I've tried again this morning and the problem was disappeared :-)

**Comment by loicsimon** [comment]
I have the same problem with IE 11 but it depends on browser width. You can find a example with a width of 1280 pixels :
![caret_bug_ie11](https://cloud.githubusercontent.com/assets/3989584/7049634/4bce2f2c-de1b-11e4-8aaa-94c1dc371587.png)

When I reduce the window to 1133 pixels, the problem disappears : 
![caret_bug_ie11-1133pixels](https://cloud.githubusercontent.com/assets/3989584/7049653/74ebcda6-de1b-11e4-90f1-6d36bc31408c.png)

If I revert the fix for issue #15682 then the caret arrow isn't splitted anymore.

**Comment by cvrebert** [comment]
I can't reproduce on IE11:
![0002screenshot](https://cloud.githubusercontent.com/assets/419884/7054592/5f3a4538-ddf2-11e4-9c8e-78877686b273.png)

@juthilo @XhmikosR Are either of you able to reproduce on IE11?

**Comment by patrickhlauke** [comment]
FWIW can't reproduce either (even tried setting monitor resolution to different sizes, and turning cleartype on/off)

**Comment by XhmikosR** [comment]
Seems to work fine here too.

**Comment by loicsimon** [comment]
Maybe the problem occurs because I'm using IE11 through an RDP connection. I'll give it a try in two weeks (i'm currently on vacation).

**Comment by patrickhlauke** [comment]
That could indeed be part of the issue ... I've seen some similar problems before with screen resolutions that aren't completely matching native screens, and some browsers then simply dropping a 1px line or mysteriously reintroducing a 1px gap somewhere)

**Comment by architshukla** [comment]
I was having the same issue today on IE. Explicitly setting a height of 5px on the span seemed to fix the issue.
```
<span style="height: 5px;" class="caret"></span>
```
Works fine on Firefox before and after doing this.

Before
![before](https://cloud.githubusercontent.com/assets/4814585/7597434/1cbb1338-f911-11e4-8bfe-8e198a054264.png)

After
![after](https://cloud.githubusercontent.com/assets/4814585/7597441/22fe896e-f911-11e4-9df9-cd08d406d1ab.png)

**Comment by XhmikosR** [comment]
I wonder if that has any side effects. If not, it seems like a good
solution.

**Comment by XhmikosR** [comment]
I mean we set height: 0 for caret; there must be some reason for this.

**Comment by architshukla** [comment]
The problem recurred today. For page zoom % set at 100% is looks fine but it starts rendering incorrectly at 105%. I did, however, notice that the CSS class for caret had a dashed border top.
```
.caret {
    ...
    border-top: 4px dashed;
    ...
}
```
Changing it to a solid border fixed the issue for me today and I reset the height to 0px. Although, it's weird that this happened only on IE. The caret looks fine on Firefox and Chrome before and after I made the change.
```
.caret {
    ...
    border-top: 4px solid;
    ...
}
```
Was there a reason to use a dashed background for creating the triangle and do you see and side effects that could stem from this?

**Comment by XhmikosR** [comment]
@mdo @cvrebert: this seems to be a rather new change in 119ce267a60732ba98513e5ed329baa3d866168a

Also, as you can see in that commit, there are more reports there.

I say we revert that change.

**Comment by cvrebert** [comment]
@XhmikosR I think we fixed this in #16258. Page zooming is a lost cause.

**Comment by XhmikosR** [comment]
If that PR fixed the issue, then I agree about zooming, it isn't important
at this point.
